### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by Howard-C**
;-)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Wouldn't the copyright strings need to get changed to `Copyright (C) 2002-2020 MuseScore BVBA and others`?

**Comment by Howard-C** [comment]
I suppose so.

**Comment by Howard-C** [comment]
To be sure, `others` or `Others`?

**Comment by Jojo-Schmitz** [comment]
I'm not sure about either. @anatoly-os ?

File: mscore/aboutbox.ui [hunk 1]
```diff
@@ -160,7 +160,7 @@ p, li { white-space: pre-wrap; }
...
 &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial'; font-size:10pt;&quot;&gt;Copyright &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;©&lt;/span&gt;&lt;span style=&quot; font-family:'Arial'; font-size:10pt;&quot;&gt; 1999-2019 Werner Schweer and Others. &lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial'; font-size:10pt;&quot;&gt;Copyright &lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;©&lt;/span&gt;&lt;span style=&quot; font-family:'Arial'; font-size:10pt;&quot;&gt; 1999-2020 MuseScore BVBA and others. &lt;/span&gt;&lt;/p&gt;
```
Comment (Jojo-Schmitz) [inline mscore/aboutbox.ui]: This string is just a placeholder, it gets set and translated programmatically [elswhere](https://github.com/musescore/MuseScore/pull/5572/commits/8d54265a5fd473f8f11e6425b321ec6ae1679dc7#diff-fd04fe0ad9c9cfe8660abedaca1a221fL119]. So changing it here is not really needed, it is OK for consistency though.

Comment (Howard-C) [reply mscore/aboutbox.ui]: Yes, I saw that ;-)

**Comment by dmitrio95** [comment]
> Wouldn't the copyright strings need to get changed to `Copyright (C) 2002-2020 MuseScore BVBA and others`?

Probably not, we shouldn't change copyright notices for previous years if only we are not adding a more correct information to them. Also MuseScore BVBA probably didn't exist in 2002 so this statement wouldn't make sense anyway. My suggestion would be to add a separate line stating something like
```
Copyright (C) 2019-2020 MuseScore BVBA and others
```
although I may also be wrong about that.

**Comment by Howard-C** [comment]
So
```
Copyright (C) 2002-2019 Werner Schweer and others
Copyright (C) 2019-2020 MuseScore BVBA and others
```
?

**Comment by Jojo-Schmitz** [comment]
I guess @ana aor @dmitrio95 may need to involve UG's legal department for clatifying this?

**Comment by dmitrio95** [comment]
So the layer's answer (at least regarding About dialog) suggests using
```
© 2002-2020 MuseScore BVBA and others
```

The "Copyright" word seems to be not needed as "©" symbol has the same meaning.

**Comment by Jojo-Schmitz** [comment]
lawyer, not layer  :-)

**Comment by dmitrio95** [comment]
Oh, yes, I meant lawyer indeed :)

**Comment by Jojo-Schmitz** [comment]
Call a lawyer a layer and he'll sue you ;-)

**Comment by Jojo-Schmitz** [comment]
BTW, the Qt copyright strings ends with `and other contributors`.
It starts with `Copyriggt (C)`, i.e. doesn't use the &copy symbol.
The MusicXML copyright though does have `Copyright &copy`

**Comment by Howard-C** [comment]
I think I've seen `Copyright ©` everywhere...

**Comment by Howard-C** [comment]
> `and other contributors`

Probably too clumsy...

**Comment by Jojo-Schmitz** [comment]
While at it, could you please change the MusicXML copyright to `...2004-2017...`? That's what their specs say.

**Comment by Jojo-Schmitz** [comment]
Also MuseScore copyright seems to start at 1999, not 2002

**Comment by dmitrio95** [comment]
> our `...2004-2018...` seems wrong

That was introduced in https://github.com/musescore/MuseScore/commit/9056e24c5b7eabbfc73bd1863402341c7d848865, might have been done mistakenly though.

> Also MuseScore copyright seems to start at 1999, not 2002

That seems strange that we have different years in license files and in "About" dialog in the application itself. Maybe it would indeed be better to keep those in sync.

**Comment by Jojo-Schmitz** [comment]
Indeed looks like a mistake, guess the MuseScore copyright date was meant at the time?

**Comment by dmitrio95** [comment]
@Howard-C could you please update this PR with these changes (1999 year in license files and `2004-2017` in "About MusicXML" dialog)?

**Comment by Jojo-Schmitz** [comment]
Better fix the UI file manually in a plain text editor, replacing that one number, most of those changes are not needed or even harmful, like the font change

File: LICENSE.rtf [hunk 2]
```diff
@@ -1,98 +1,98 @@
...
+\par
+12. IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING WILL ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY MODIFY AND/OR REDISTRIBUTE THE PROGRAM AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES, INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE THE PROGRAM (INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.\par
+\par
+END OF TERMS AND CONDITIONS\par
+}
+
```
Comment (Howard-C) [inline LICENSE.rtf]: The encoding was changed from ANSI to UTF-8 when edited with Notepad.

**Comment by dmitrio95** [comment]
@Howard-C, @Jojo-Schmitz, thanks for taking care of this update! :)

**Pull request merged**